presto-native-execution/src/test/java/com/facebook/presto/nativeworker/HiveExternalWorkerQueryRunner.java

```diff
@@ -34,7 +35,7 @@ public static void main(String[] args)
         javaQueryRunner.close();
 
         // Launch distributed runner.
-        DistributedQueryRunner queryRunner = (DistributedQueryRunner) PrestoNativeQueryRunnerUtils.createQueryRunner(false, false, false, false);
+        DistributedQueryRunner queryRunner = (DistributedQueryRunner) PrestoNativeQueryRunnerUtils.createQueryRunner(false, false, false, false, ImmutableMap.of());
```

Comment:
https://github.com/prestodb/presto/pull/25120 for QueryRunner with builders should soon be submit. Please refactor your code according to that.

Reply:
Thanks, done!